### DISCUSSION TITLE
Synology docker Vaultwarden - smtp problem

### DISCUSSION THREAD

**Discussion by sadden3194** [dani-garcia/vaultwarden#2302] [General]
Hi !

I would like to configure the mail function.
I tried this:

![62115-a274627eeedaa5960aaf315102df4a9d](https://user-images.githubusercontent.com/17011498/153700980-9f36ee4a-5580-48c2-9c90-d3d1d0dff024.jpg)

But I get this error:
![62114-e77d2c444c541f279c2e3aee552c03f3](https://user-images.githubusercontent.com/17011498/153700988-e8eee0a9-b4ef-4181-92ef-a1d80e5e25a7.jpg)

How can I solve this?

**Comment by Masgalor** [comment]
See #2249
